import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/AMPStaticProcessor.java

```diff
@@ -863,21 +889,27 @@ private Object getMultilangString(InternalDocument source, String destinationFie
 					String provider = StringUtils.isBlank(value.get("providing-org"))
 							? organization.getValue().get("value") : value.get("providing-org");
 
+					if(provider == null){
+						break;
+					}
+					//TODO we should search organizations by ref and no by value
+					provider = provider.toUpperCase(); 
 
 					List<FundingDetail> fundingDetails = providerFundingDetails
 							.getOrDefault(provider, new ArrayList<FundingDetail>());
 
 					FundingDetail fd = new FundingDetail();
 					fd.setTransactionType(tTNameSourceMap.get(destinationSubType));
 					fd.setTransactionAmount(getTransactionAmount(value.get("value"), percentage));
-					fd.setTransactionDateTimeStamp(isTransactionDateTimeStam("fundings~"+ sourceField.getDisplayName().toLowerCase() + "~transaction_date"));
+					fd.setTransactionDateTimeStamp(isTransactionDateTimeStamp("fundings~"+ sourceField.getDisplayName().toLowerCase() + "~transaction_date"));
```

Comment:
I know this is old code but could we have a follow up ticket to move all these hardcoded strings into reusable constants?